### PR TITLE
Hs 2360

### PR DESCRIPTION
**Pull request by aslobodeniuk**

### CODE REVIEW

File: gst/audioparsers/put_bits.h [hunk 1]
```diff
@@ -223,6 +223,7 @@ static inline void put_sbits(PutBitContext *pb, int n, int32_t value)
 /**
  * Write exactly 32 bits into a bitstream.
  */
+#if 0
```
Comment (ylatuya) [inline gst/audioparsers/put_bits.h]: Maybe defining av_unused to  [G_GNUC_UNSED ](https://developer.gnome.org/glib/stable/glib-Miscellaneous-Macros.html#G-GNUC-UNUSED:CAPS) can help

Comment (aslobodeniuk) [reply gst/audioparsers/put_bits.h]: thanks, it worked ! Adding -Wall -Werror to compiler flags..

Comment (aslobodeniuk) [reply gst/audioparsers/put_bits.h]: oh, I was writing about a bit different issue - last warning.

Comment (aslobodeniuk) [reply gst/audioparsers/put_bits.h]: fixed

**Review by ylatuya — APPROVED** [review]

**Pull request merged**